### ISSUE TITLE
[EPIC] Branding

### ISSUE THREAD

**Issue by VitaliyKulikov** [extesy/DeckTracker#89]
Good news that I see that more and more popular **Twitch**-streamers are using **DeckTracker**.
It will be nice opportunity to crowdsource and increase users base. To cover this it will be nice to:
- **DeckTracker** should be better to remember then concurrents 
- **DeckTracker** will be easy to find and install
- **DeckTracker** should be involved in any games activity (#88)

Just quick-list of possible improvements:
- logo at self and visible one on overlay in-game
- think about change name 
- website (#30)

**Comment by extesy** [comment]
Agreed with everything and it was all in my mental plan already :). Regarding the name - I called it like that because the architecture makes it relatively easy to add support for any other Unity-based CCG (Shadowverse, Faeria, Duelyst, etc). I don't want to stop on only two DWD's games - these were just the beginning. The only thing I need to support more games is time. So, what do you think is a better name?

As for website - I've created a project locally (asp.net core with react/redux) but haven't published it yet because it's pretty much empty and I'm still learning react. Do you know web development? If you do then I can publish it as-is and you can help creating a web site.

**Comment by VitaliyKulikov** [comment]
More games, More fun - it's good )) Also, for more games linux support will be needed #22 (Faeria is awesome).

Names: 
- [LadyBird ](http://4kwallpapershd.com/wp-content/uploads/2016/08/Cool-Ladybug-Large-Image.jpg)
- [Dame](https://en.wikipedia.org/wiki/Dame)
- **Debo** <- _**De**ck **Bo**ard_ | **Detra** <- _**De**ck **Tra**cker_
- Duni | Trout | Von
more later ))

Website: I am mostly Enterprise Java-stack based and server side guru )) So, for web-development, front-side is not my powerfull skill. Can help on client-server stuff )) But, let's see - it should not be so complicated. 

Have you tried AngularJS ? ))
